pkgs/games/steam/chrootenv.nix

```diff
@@ -37,7 +38,8 @@ let
       # Zoneinfo
       etc-zoneinfo
     ] ++ lib.optional withJava jdk
-      ++ lib.optional withPrimus primus2;
+      ++ lib.optional withPrimus primus2
+      ++ extraPkgs;
```

Comment:
And here pass `pkgs` to the function: `extraPkgs pkgs`.

This way you will use the same package set `buildFHSEnv` passes to the function above.